compose/acq-pod/launch_bluesky.sh

```diff
@@ -30,9 +50,9 @@ podman run --pod pod_acq-pod  \
        --network-alias sneaky \
        -ti  --rm \
        -v /tmp/.X11-unix/:/tmp/.X11-unix/ \
-       -e DISPLAY \
-       -v $XAUTH:$XAUTH \
-       -e XAUTHORITY=$XAUTH \
+       -e DISPLAY=$LOCAL_DISPLAY \
+       -v $XAUTH:/tmp/.docker.xauth \
```

Comment:
I don't see how this works on mac as $xauth is someplace different?

Reply:
The previous version used `$XAUTH` as both arguments for the mount and it was assumed that the location on the host `/tmp/.docker.xauth` would be the same as in the container. 

This just removes that implicit assumption, and explicitly specifies a location in the container.

Reply:
ooh, sorry miss-reading mount incantations.